src/Uno.UI/UI/Xaml/Input/FocusManager.cs

```diff
@@ -20,7 +20,6 @@ namespace Windows.UI.Xaml.Input
 	public sealed partial class FocusManager
 	{
 		private static readonly Lazy<Logger> _log = new Lazy<Logger>(() => typeof(FocusManager).Log());
-		private static readonly Dictionary<XamlRoot, object> _focusedElements = new Dictionary<XamlRoot, object>(1);
```

Comment:
@MartinZikmund

Reply:
Ok, it was superseeded by the proper focus manager implementation.